### PR TITLE
Calypsoify: Update Colours to the New Palette

### LINKED ISSUE #36098: Update Calypsoify to the New Color Palette
Following #35949, all Calypsoify [should be updated](https://github.com/Automattic/wp-calypso/pull/35949#issuecomment-528515405) to the most recent Color Studio colors.

### PR DESCRIPTION
**Pull request by Aurorum**
Fixes Automattic/wp-calypso#36098

#### Changes proposed in this Pull Request:

* Updates Calypsoify to use the more recent colour palette, which was merged to WordPress.com in Automattic/wp-calypso#35949

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Changes an existing feature

#### Testing instructions:

* Go to a Calypsoified page and compare the colours 

**Before:**
Screenshot 2019-10-02 at 18 59 32

**After:**
screenshot

Verify that the colours look good and ultimately, match WordPress.com. You can find the WordPress.com equivalents here: https://github.com/Automattic/wp-calypso/blob/master/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss

#### Proposed changelog entry for your changes:

* Calypsoify: update the colour scheme to the most recent palette.

### CODE REVIEW

**Comment by jeherve** [comment]
> I've uploaded it to the file itself as a placeholder, but is there an alternative?

You can use this link instead of uploading the screenshot to the repo:
https://cldup.com/awmrHOWz7t.png

Thanks!

**Comment by Aurorum** [comment]
Thanks @jeherve, updated to use that link instead. :)

Marking this as ready for review.

**Review by jeherve — APPROVED** [review]
This looks good to me, but I'll let a designer chime in with this. 👍

**Review by keoshi — APPROVED** [review]
Looks good to me! 🛳

File: modules/calypsoify/_studio-wpcom.scss [hunk 1]
```diff
@@ -0,0 +1,172 @@
+// v2.2.0
```
Comment (simison) [inline modules/calypsoify/_studio-wpcom.scss]: As a follow up I think we can import these from color studio directly

`@import '~@automattic/color-studio/path_to_the_file';`

**Pull request merged**